src/website/src/styles/global.css

```
@@ -343,36 +343,98 @@ pre code,
+ * uniform --pixel-frame-wide pixel border on all four corners. Reusing a
+ * reduced step there would pinch the border to zero width exactly at each
+ * notch's reflex vertex. ::after draws an inset groove line (the "bezel")
+ * that follows the same inner stepped silhouette.
+ */
 .pixel-card {
-  background: var(--color-surface);
-  border: 2px solid var(--color-border);
-  padding: var(--space-xl);
+  --pixel-step: 14px;
+  --pixel-frame: 6px;
   position: relative;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)